Lib/unittest/mock.py

```diff
@@ -98,6 +98,12 @@ def _get_signature_object(func, as_instance, eat_self):
         func = func.__init__
         # Skip the `self` argument in __init__
         eat_self = True
+    elif isinstance(func, classmethod | staticmethod):
```

Comment:
```suggestion
    elif isinstance(func, (classmethod,  staticmethod)):
```

We should stick to tuples in this case :)